src/views/demoDetails/PlayerList/index.tsx

```diff
@@ -35,13 +37,34 @@ const useStyles = createStyles((theme) => ({
     display: "inline-block",
     verticalAlign: "top",
   },
+  scoreboardDivider: {
+    border: "none",
+    borderTop: "1px solid #444",
+    margin: "16px",
+  },
+  scoreboardPlayerNameHeader: {
+    paddingLeft: "16px",
+    fontWeight: 800,
+    fontSize: "16pt",
+  },
```

Comment:
Instead of hardcoding font size/weight, we could use the [Text](https://mantine.dev/core/text/) or [Title](https://mantine.dev/core/title/) component which will be consistent with other parts of the app.